cube/src/test/java/zone/cogni/asquare/cube/sort/StatementSorterTest.java

```diff
@@ -3,69 +3,165 @@
 
 import com.google.common.collect.ImmutableMap;
 import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.Property;
+import org.apache.jena.rdf.model.RDFNode;
+import org.apache.jena.rdf.model.Resource;
 import org.apache.jena.rdf.model.Statement;
+import org.apache.jena.vocabulary.RDF;
 import org.junit.jupiter.api.Test;
 import org.springframework.core.io.ClassPathResource;
-import zone.cogni.sem.jena.JenaUtils;
+import zone.cogni.asquare.cube.digest.SortedBlock;
+import zone.cogni.libs.jena.utils.JenaUtils;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
+import static org.apache.jena.rdf.model.ResourceFactory.createStringLiteral;
 import static org.assertj.core.api.Assertions.assertThat;
 
 class StatementSorterTest {
 
+
```

Comment:
lots of new and much simpler tests... those mixed tests were actually already to complex to correctly evaluate